### ISSUE TITLE
The parameter "templating.helper.code.file_link_format" has a dependency on a non-existent parameter "f&line="

### ISSUE THREAD

**Issue by Perf** [symfony/symfony#31431]
**Symfony version(s) affected**: 4.3@dev

**Description**  
Having this issue after upgrading from 4.2 to 4.3@dev

**How to reproduce**  
1. ensure `php.ini` contains:
```ini
xdebug.file_link_format = "phpstorm://open?file=%f&line=%l&/docker/path/to/app>/host/path/to/app"
```
2. execute in running container with app :
```bash
bin/console debug:container
```
3. get an Exception
> In ParameterBag.php line 100:
> The parameter "templating.helper.code.file_link_format" has a dependency on a non-existent parameter "f&line=".

**Possible Solution**  
¯\\\_(ツ)\_/¯

**Additional context**  
`framework.ide` parameter is **not** configured in SF config.

**Comment by ro0NL** [comment]
i believe this will be fixed in #30930

**Comment by Perf** [comment]
Perfect, seems to be so!
Thanks @ro0NL for update!
